integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/l0promotion/InternalSdxSshAndCmAccessTest.java

```diff
@@ -0,0 +1,144 @@
+        String recipeName = resourcePropertyProvider().getName();
+        String stack = resourcePropertyProvider().getName();
+        String filePath = "/post-install";
+        String fileName = "post-install";
+        String masterInstanceGroup = "master";
+        String idbrokerInstanceGroup = "idbroker";
+        String newWorkloadPassword = "Admin123@";
+        String workloadUsername = getWorkloadUsername(testContext);
+
+        testContext
+                .given(clouderaManager, ClouderaManagerTestDto.class)
+                .given(cluster, ClusterTestDto.class).withBlueprintName(getDefaultSDXBlueprintName()).withValidateBlueprint(Boolean.FALSE)
+                .withClouderaManager(clouderaManager)
+                .given(RecipeTestDto.class).withName(recipeName).withContent(generateRecipeContent())
+                .withRecipeType(POST_CLOUDERA_MANAGER_START)
+                .when(recipeTestClient.createV4())
+                .given(masterInstanceGroup, InstanceGroupTestDto.class).withHostGroup(MASTER).withNodeCount(1).withRecipes(recipeName)
+                .given(idbrokerInstanceGroup, InstanceGroupTestDto.class).withHostGroup(IDBROKER).withNodeCount(1).withRecipes(recipeName)
+                .given(stack, StackTestDto.class).withCluster(cluster).withInstanceGroups(masterInstanceGroup, idbrokerInstanceGroup)
+                .given(sdxInternal, SdxInternalTestDto.class)
+                .withCloudStorage(getCloudStorageRequest(testContext))
+                .withStackRequest(key(cluster), key(stack))
+                .when(sdxTestClient.createInternal(), key(sdxInternal))
+                .await(SdxClusterStatusResponse.RUNNING)
+                .awaitForInstance(getSdxInstancesHealthyState())
+                .then((tc, testDto, client) -> sshJUtil.checkFilesOnHostByNameAndPath(testDto, client, List.of(MASTER.getName(), IDBROKER.getName()),
+                            filePath, fileName, 1, null, null))
+                .then((tc, testDto, client) -> clouderaManagerUtil.checkClouderaManagerKnoxIDBrokerRoleConfigGroups(tc, testDto, client))
+                .validate();
+
+        testContext
+                .given(freeIpa, FreeIpaTestDto.class)
+                .when(freeIpaTestClient.describe(), key(freeIpa))
+                .given(FreeIpaUserSyncTestDto.class)
+                .when(freeIpaTestClient.getLastSyncOperationStatus())
+                .await(OperationState.COMPLETED)
+                .given(UmsTestDto.class).assignTarget(EnvironmentTestDto.class.getSimpleName())
+                .when(umsTestClient.setWorkloadPassword(newWorkloadPassword))
+                .given(FreeIpaUserSyncTestDto.class)
+                .when(freeIpaTestClient.syncAll())
+                .await(OperationState.COMPLETED)
+                .given(sdxInternal, SdxInternalTestDto.class)
+                .then((tc, testDto, client) -> sshJUtil.checkFilesOnHostByNameAndPath(testDto, client, List.of(MASTER.getName()),
```

Comment:
this file check is duplicated, I would do it only once

Reply:
In this second try we SSH with user and password (namely with workload username and updated workload password). If this is success, that means our test user also can SSH to the `MASTER` with its new workload password.

Reply:
makes sense, thank you!